src/main/java/com/djrapitops/extension/GriefDefenderExtension.java

```diff
@@ -88,27 +92,23 @@ public long claimCount(UUID playerUUID) {
     @StringProvider(
             text = "Claim Type",
             description = "Basic + Town + SUBDIVISION + Admin",
-            iconName = "map-marker",
+            iconName = "sign",
             iconColor = Color.BLUE,
             showInPlayerTable = true
     )
     public String claimType(UUID playerUUID) {
-        if (getClaimsOf(playerUUID) != null) {
-            getClaimsOf(playerUUID)
-                    .forEach(this::countType);
+        int basic=0, town=0, sub=0, admin=0;
+        List<Claim> claimsList = getClaimsOf(playerUUID);
+        for (Claim claim : claimsList) {
+            if (claim.isBasicClaim()) {
+                basic++;
+            } else if (claim.isTown()) {
+                town++;
+            } else if (claim.isSubdivision()) {
+                sub++;
+            } else admin++;
         }
-        return b + "+" + t + "+" + s + "+" + a ;
-    }
-
-    int b=0, t=0, s=0, a=0;
-    public void countType(Claim claim) {
-        if (claim.isBasicClaim()) {
-            b++;
-        } else if(claim.isTown()) {
-            t++;
-        } else if(claim.isSubdivision()) {
-            s++;
-        } else a++;
+        return basic + "+" + town + "+" + sub + "+" + admin ;
```

Comment:
Does the following make sense to a user?

> Claim Type: 5+4+1+3

I think it is a bit confusing.
You might want to split this into 4 separate StringProviders or name what the numbers are in the String